### PR TITLE
Release docs for v0.8.0.

### PR DESCRIPTION
**Pull request by taylortom**
TODO.

(below generated using tool at https://aat-releaser.herokuapp.com/).

### CODE REVIEW

**Comment by canstudios-nicolaw** [comment]
@taylortom Would you like me to do anything with this?

**Comment by taylortom** [comment]
@canstudios-nicolaw not really, just to check through and sign off really 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ All notable changes to the Adapt authoring tool are documented in this file.
 **IMPORTANT**: For information on how to **correctly and safely** update your installation, please consult **INSTALL.md**.<br/>
 _Note that we adhere to the [semantic versioning](http://semver.org/) scheme for release numbering._
 
+## [0.8.0] - 2019-05-01
```
Comment (tomgreenfield) [inline CHANGELOG.md]: Please bump to today's date.

Comment (taylortom) [reply CHANGELOG.md]: no way

Comment (canstudios-nicolaw) [reply CHANGELOG.md]: @tomgreenfield want me to update this to today's date now?

**Pull request merged**